python/test/t_NAIS_std.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+
+"""
+Non parametric Adaptive Importance Sampling algorithm
+Python implementation of adaptive Importance Sampling by Cross Entropy
+M. Balesdent and L. Brevault of ONERA, the French Aerospace Lab for the
+openTURNS consortium
+
+source :  J. Morio & M. Balesdent, Estimation of Rare Event Probabilities
+in Complex Aerospace and Other Systems, A Practical Approach, Elsevier, 2015
+
+
+"""
+
+# Parameters of the class
+# event : ThresholdEvent based on composite vector of input variables on limit state function
+# n_IS : number of IS samples at each step (integer)
+# rho_quantile : percentage of points that are in the local failure domain (float between 0 and 100)
+
+import numpy as np
```

Comment:
the mingw build do not have numpy installed so this test fails, you can use sqrt from the math module instead